framework/src/modules/chain/blocks/blocks.js

```diff
@@ -302,6 +304,26 @@ class Blocks extends EventEmitter {
 
 	// Process a block from the P2P
 	async receiveBlockFromNetwork(block) {
+		this.logger.info(
```

Comment:
Blocks module is too low level to have `info` logs.
Within module, it should be maximum `debug`

Reply:
I will change info logs to debugs for new code, however refactored and legacy code still uses info in this file.